resolve/testdata/resolve.star

```diff
@@ -235,6 +235,123 @@ def g(args, b, *args): pass ### "duplicate parameter: args"
 def h(kwargs, a, **kwargs): pass ### "duplicate parameter: kwargs"
 def i(*x, **x): pass ### "duplicate parameter: x"
 
+---
```

Comment:
I don't think the test carries its weight. It's more complex than the logic under test, and a spec-compliant compiler is not required to report an error (indeed should not). Also, 255 args should be enough for anyone. :)

Reply:
Reverted. It was useful for me during development to make sure I got all the corner cases right (and led to #83), but I agree, it's an eyesore.